ml_ops.sh

```diff
@@ -1,13 +1,20 @@
 #!/bin/bash
 
 # read in variables (except for date) from etc/.conf file
-
 FDATE=$1
-YR=$2
-MH=$3
-DY=$4
-DSOURCE=$5
-TOL=$6
+DSOURCE=$2
+TOL=$3
+YR=${FDATE:0:4}
+MH=${FDATE:4:2}
+DY=${FDATE:6:2}
+
+# checking for required arguments
+if [[ "${#FDATE}" != "8" || -z "${DSOURCE}" ]]; then
+    echo "ml_ops.sh syntax error"
+    echo "Please run ml_ops.sh again with the correct syntax:"
+    echo "./ml_ops.sh 19700101 flow"
```

Comment:
in example, use a date in which it is clear that the order is YYYYMMDD

eg.

19700522